src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/repositories/ReportRepository.kt

```diff
@@ -119,6 +121,20 @@ class OrderlyReportRepository(val isReviewer: Boolean,
         }
     }
 
+    override fun getReportVersionInstance(version: String): Map<String, String>
```

Comment:
As this returns multiple instances, should probably be named `getReportVersionInstances`